apps/web/pages/[user].tsx

```diff
@@ -391,10 +392,16 @@ export const getServerSideProps: GetServerSideProps<UserPageProps> = async (cont
 
   // if profile only has one public event-type, redirect to it
   if (eventTypes.length === 1 && context.query.redirect !== "false") {
+    const {
+      // So it doesn't display in the Link
+      user: _user,
+      ...query
+    } = context.query;
+
     return {
       redirect: {
         permanent: false,
-        destination: `/${user.username}/${eventTypes[0].slug}`,
+        destination: `/${user.username}/${eventTypes[0].slug}?${stringify(query)}`,
```

Comment:
I don't think we want this. Prefilling is already supported for event types url /[user]/[type].